plugins/database/src/low_level_odbc.cpp

```diff
@@ -313,14 +313,14 @@ cllCheckPending( const char *sql, int option, int dbType ) {
             }
         }
 
-        log_db::trace("Warning, pending SQL at cllDisconnect, count: {}", pendingCount);
+        log_db::info("{}: Warning, pending SQL at cllDisconnect, count: {}", __func__, pendingCount);
```

Comment:
Hmm, we use `info`, but the message contains `Warning`. Let's change this to the following:
```cpp
log_db::warn("{}: Pending SQL ...
```

Reply:
Done.